packages/google_sign_in/google_sign_in_platform_interface/lib/google_sign_in_platform_interface.dart

```diff
@@ -121,4 +121,9 @@ abstract class GoogleSignInPlatform {
   Future<void> clearAuthCache({@required String token}) async {
     throw UnimplementedError('clearAuthCache() has not been implemented.');
   }
+
+  /// Requests the user grants additional Oauth [scopes].
+  Future<bool> requestScopes(List<String> scopes) async {
```

Comment:
what can be the values of the `scopes`? Maybe we should either make an enum or have some concrete list of what the values can be.

Reply:
@cyanglaz There's a TON of auth scopes, here's the list: https://developers.google.com/identity/protocols/googlescopes (each API that you want to use has its own set of scopes)

Reply:
Yes, that's a good point! It can probably be added to the README.md.

Reply:
Added a link in the dartdoc. Looks like this is already mentioned in the README.md [here](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/google_sign_in/README.md#use-the-plugin).